tests/event_processor_test.php

```diff
@@ -1153,8 +1153,11 @@ public function test_cleanup_history() {
         $DB->execute('UPDATE {tool_trigger_run_hist} SET executed = :veryold', ['veryold' => 12345]);
 
         // Run the task. It should delete this record from history.
+        // Output buffering is used to eat the output. The record inspection is the tested component.
         $task = new \tool_trigger\task\cleanup_history();
+        ob_start();
```

Comment:
If this because debugging is called ? If so, `$this->resetDebugging` is probably better suited. Otherwise if its outputting html then the current code is fine

Reply:
Not relevant - its just mtrace causing the test to be risky